docs/helpers/ObjectSerializer.md

```diff
@@ -0,0 +1,81 @@
+title: Object Serializer
+author: simop
+description: IObjectSerializer is an interface that you can implement to provide a serializer of your choice to ObjectStorageHelper.
+keywords: windows 10, uwp, windows community toolkit, uwp community toolkit, uwp toolkit, serialization
+dev_langs:
+  - csharp
+  - vb
+---
+
+# ObjectSerializer
+
+You should implement IObjectSerializer when you need to write data using this toolkit's helpers with a custom serializer. If you don't, a default Json serializer will be used otherwise.
+
+## Methods
+
+| Methods | Return Type | Description |
+|---------|-------------|-------------|
+| Serialize<T>(T value)        | string | Serialize an object of type T into a string. |
+| Deserialize<T>(string value) | T      | Deserialize a string to an object of type T. |
+
+## Examples
+
+### Json.NET
+
+```csharp
+using Microsoft.Toolkit.Uwp.Helpers;
+using Newtonsoft.Json;
+
+namespace Contoso.Helpers
+{
+    public class JsonNetObjectSerializer : IObjectSerializer
+    {
+        // Specify your serialization settings
+        private readonly JsonSerializerSettings settings = new JsonSerializerSettings();
+
+        public string Serialize<T>(T value) => JsonConvert.SerializeObject(value, typeof(T), Formatting.Indented, settings);
+
+        public T Deserialize<T>(string value) => JsonConvert.DeserializeObject<T>(value, settings);
+    }
+}
+```
+
+### DataContract
+
+```csharp
+using System.IO;
+using System.Runtime.Serialization;
+using System.Xml;
+
+namespace Contoso.Helpers
+{
+    public class DataContractObjectSerializer : IObjectSerializer
+    {
+        // Specify your serialization settings
+        private readonly DataContractSerializerSettings settings = new DataContractSerializerSettings();
+
+        public string Serialize<T>(T value)
+        {
+            var serializer = new DataContractSerializer(typeof(T), settings);
+
+            using (var stringWriter = new StringWriter())
+            using (var xmlWriter = XmlWriter.Create(stringWriter))
+            {
+                serializer.WriteObject(xmlWriter, value);
+                return stringWriter.ToString();
+            }
+        }
+
+        public T Deserialize<T>(string value)
+        {
+            var serializer = new DataContractSerializer(typeof(T), settings);
+
+            using (var stringReader = new StringReader(value))
+            using (var xmlReader = XmlReader.Create(stringReader))
+            {
+                return serializer.ReadObject(xmlReader) as T;
+            }
+        }
+    }
+}
+```

```

Comment:
Should follow the template here and have the OS requirement info and the link to source code in the repo.